### DISCUSSION TITLE
I'd create a light version the SaaS-Boilerplate

### DISCUSSION THREAD

**Discussion by mingolladaniele** [apptension/saas-boilerplate#444] [Ideas]
Hello team!
Let's start by say thank you for all your incredible work. I think that the SaaS-Boilerplate is really a fantastic project!

I've only found one problem: the SaaS-Boilerplate is good for mid-senior people and it contains so many features that for most startup are not usefull.

I've cloned the repo 3 months ago and I'm still understanding how does it work the entire project, how to fix the issues here and there and so on.

I'll suggest to create another boilerplate based on this one but with the only features really needed (CRUD, Postgres, etc.)

Thanks!

**Comment by derevyan** [comment]
Don't find it in your repos. Will be happy to participate on this work.
